### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by a32543254**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by kevinintel — APPROVED** [review]

**Review by hshen14 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -212,6 +212,27 @@ model = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True)
...
 ```
 
+You can also load your quantized low bit model(GPTQ/AWQ/RTN/AutoRound) by the below code.
+```python
+from transformers import AutoTokenizer, TextStreamer
+from intel_extension_for_transformers.transformers import AutoModelForCausalLM, WeightOnlyQuantConfig
+
+# Download Hugging Face GPTQ/AWQ/AutoRound model to local path
+model_name = "PATH_TO_MODEL"  # local path to model
+woq_config = WeightOnlyQuantConfig(use_gptq=True)   # use_awq=True for awq model
```
Comment (kevinintel) [inline README.md:222]: autoround?

Comment (a32543254) [reply README.md:222]: Ok, I will add auto round args

**Review by kevinintel — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hshen14) [reply README.md]: Download Hugging Face GPTQ/AWQ model or generate model to local

Comment (a32543254) [reply README.md]: fixed on https://github.com/intel/intel-extension-for-transformers/pull/1196/commits/b12655d8247d93b81cd5297655badedf687ccbfd

**Pull request merged**